src/resources/books.md

```diff
@@ -1,14 +1,14 @@
 ---
 layout: default
 permalink: /resources/books
-title: Books about Dart
+title: Books and Videos about Dart
 description: Read all about it! Here's a collection of books about Dart.
 toc: false
 ---
 
-Here's a collection of books about Dart.
+Here's a collection of books and videos about Dart.
 [Let us know](https://github.com/dart-lang/site-www/issues)
-if you find another book that we should add.
+if you find another resource that we should add.
```

Comment:
My recent dive into the style guide has informed me that we should almost always put the qualifier before the "do this" thing. So:

If you ...., [let us know](...).